packages/calcite-components/src/components/shell/shell.scss

```diff
@@ -78,7 +73,8 @@
 
 slot[name="panel-start"]::slotted(calcite-shell-panel),
 slot[name="panel-end"]::slotted(calcite-shell-panel) {
-  @apply relative;
+  @apply relative
+    z-default;
   z-index: calc(var(--calcite-z-index) + 1);
```

Comment:
It looks like the `z-default` util is overridden here. Can you confirm?

Reply:
Oh, nice catch. Will fix